README.md

```diff
@@ -71,111 +81,107 @@ export class AppModule {}
-| ----------- | ------------------------------------------------  | ----------------------------------------------------------------------------------------- |
-| nodeMoved   | [NgcxTreeNodeMovedEvent](#NgcxTreeNodeMovedEvent) | fired when a node is moved                                                                |
-| customEvent | any                                               | may be fired by your own custom component                                                 |
-| clickEvent  | [NgcxTreeNodeWrapper](#NgcxTreeNodeWrapper)       | fired when node is clicked                                                                |
-| selectEvent | [NgcxTreeNodeWrapper](#NgcxTreeNodeWrapper)       | fired when node is selected or un-selected. Clicking a selected node un-selects the node. |
+|   Property    |                 event content type                  |                                        Description                                        |
+| ------------- | --------------------------------------------------- | ----------------------------------------------------------------------------------------- |
+| `nodeMoved`   | [`NgcxTreeNodeMovedEvent`](#ngcxtreenodemovedevent) | fired when a node is moved                                                                |
+| `customEvent` | any                                                 | may be fired by your own custom component                                                 |
+| `clickEvent`  | [`NgcxTreeNodeWrapper`](#ngcxtreenodewrappert)       | fired when node is clicked                                                                |
+| `selectEvent` | [`NgcxTreeNodeWrapper`](#ngcxtreenodewrappert)       | fired when node is selected or un-selected. Clicking a selected node un-selects the node. |
 
-# Model
+## Model
 
-## NgcxTreeConfig
+### `NgcxTreeConfig`
 
 The component includes a model called `NgcxTreeConfig` with some basic optional
-settings.
+settings:
 
 - `allowDrag` method that decides if a node can be dragged:
   `(node: NgcxTreeNodeWrapper<T>) => boolean` - all nodes are draggable by
-  default<br><br>
+  default
 - `allowDrop` method that decides if node can be dropped into another node
-  `(node: NgcxTreeNodeWrapper<T>, intoNode?: NgcxTreeNodeWrapper<T>) => boolean ` -
-  every node may be draggable everywhere by default<br><br>
+  `(node: NgcxTreeNodeWrapper<T>, intoNode?: NgcxTreeNodeWrapper<T>) => boolean` -
+  every node may be draggable everywhere by default
 - `allowSelection` method that decides if node can be selected
-  `(node: NgcxTreeNodeWrapper<T>) => boolean ` - nodes are not selectable by
-  default<br><br>
+  `(node: NgcxTreeNodeWrapper<T>) => boolean` - nodes are not selectable by
+  default
 - `treeNodeContentTemplate` Angular TemplateRef that will be used to render a
-  node<br><br> `let-nodeWrapper="nodeWrapper"` may be used to access the node
+  node
+- `let-nodeWrapper="nodeWrapper"` may be used to access the node
```

Comment:
@mn-42 did I understood it correctly, that `let-nodeWrapper` should be an separate point in this list?

Reply:
should be a sub-point of treeNodeContentTemplate